builder/main.go

```diff
@@ -23,7 +23,7 @@ func main() {
 
 	runner := buildpackrunner.New(&config)
 
-	err := runner.Run()
+	_, err := runner.Run()
```

Comment:
The return value was used by our [multi-buildpack](https://github.com/cloudfoundry-incubator/multi-buildpack) in order for to find the staging_info.yml file to grab the start command.

We can probably remove the return value and change the buildpack to use a glob